libs/vscode/tasks/src/lib/cli-task-commands.ts

```diff
@@ -67,6 +75,20 @@ export function registerCliTaskCommands(
       selectCliCommandAndPromptForFlags('run', await getCliProjectFromUri(uri))
     );
 
+    commands.registerCommand(`${cli}.move.fileexplorer`, async (uri: Uri) => {
+      const getCorrectMoveGenerator = (uri: Uri) => '@nrwl/workspace:move';
```

Comment:
We could probably find out the minimum version of Nx that introduced this generator and have that logic here. We have this function here that could be exported and used outside that project:
https://github.com/nrwl/nx-console/blob/16e315521dbd90de3040abc71c28826fba44af8e/libs/vscode/nx-workspace/src/lib/nx-version.ts#L7

Reply:
Oooh, hadn't considered that. Good point.

For versions younger than we had the move/remove generators, we should probably remove this option from the context menu, right?

Reply:
Similar note: for when the user right-clicks on a node in the root, my solution would open the webview for the `@nrwl/workspace:move` generator; without a project set.

I think this is problematic, since we *should* be using the `@nrwl/angular:move` generator if the project is angular :\